win32ss/gdi/eng/mdevobj.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * COPYRIGHT:        See COPYING in the top level directory
+ * PROJECT:          ReactOS kernel
+ * PURPOSE:          Support for meta devices
+ * FILE:             win32ss/gdi/eng/mdevobj.c
+ * PROGRAMER:        Hervé Poussineau
+ */
+
+#include <win32k.h>
+#define NDEBUG
+#include <debug.h>
+DBG_DEFAULT_CHANNEL(EngMDev);
+
+PMDEVOBJ gpmdev = NULL; /* FIXME: should be stored in gpDispInfo->pmdev */
+
+VOID
+MDEVOBJ_vEnable(
+    _Inout_ PMDEVOBJ pmdev)
+{
+    ULONG i;
+
+    for (i = 0; i < pmdev->cDev; i++)
+        PDEVOBJ_vEnableDisplay(pmdev->dev[i].ppdev);
+}
+
+BOOL
```

Comment:
For LDEV functions you used BOOLEAN for returned values, but here (and the new PDEV functions) you use BOOL?

Reply:
Changed LDEV functions to BOOL